### DISCUSSION TITLE
Running locally

### DISCUSSION THREAD

**Discussion by DavidTCarpenters** [microsoft/AzUrlShortener#504] [Q&A] [answered]
Hi,

I've got a large amount of dev done on this project in a branch from an older version before the SWA was introduced, which I'm looking to port over to here and submit some PRs.

There is some work I need to test in relation to authentication and that needs to be done on my local dev machine.

How does one get this project to run locally in debug? Running swa build and swa start seems to fail with:

[swa] WARNING: Failed to load staticwebapp.config.json schema. Continuing without validation!
[swa] - Waiting for http://localhost:5000 to be ready
[swa] node "C:\Users\[redacted]\AppData\Roaming\npm\node_modules\@azure\static-web-apps-cli\dist\msha\server.js" exited with code 0

**Comment by fboucher** [comment, marked as answer]
@DavidTCarpenters the next version is available in PR #506 I would suggest using this as base for your modification or upgrades. Note that the PR is still in progress but the bigger changes are done.

**Reply by DavidTCarpenters** [reply]
Thanks Frank, I'll check it out. That might make things interesting as I'd done a lot of work to abstract the Azure Table Storage bits out and upgraded to Azure.Data.Tables
